circuits/verifier/src/rapidsnark.rs

```diff
@@ -0,0 +1,13 @@
+use crate::{traits::Verifier, verifier_from_contents};
+
+pub struct Rapidsnark;
+
+impl Verifier for Rapidsnark {
+    fn verify(
+        verification_key_contents: &[u8],
+        public_contents: &[u8],
+        proof_contents: &[u8],
+    ) -> std::io::Result<bool> {
```

Comment:
This should not be io, it should be a custom result. This is a library, the underlaying implementation may use io, or may not.